app/controllers/sidebar_sections_controller.rb

```diff
@@ -100,6 +111,31 @@ def reorder_params
 
   private
 
+  def reset_community(community_section)
+    community_section.update!(title: "Community")
+    community_section.sidebar_section_links.destroy_all
+    community_urls =
+      SidebarUrl::COMMUNITY_SECTION_LINKS.map do |url_data|
+        "('#{url_data[:name]}', '#{url_data[:path]}', '#{url_data[:icon]}', '#{url_data[:segment]}', false, now(), now())"
+      end
+
+    result = DB.query <<~SQL
+      INSERT INTO sidebar_urls(name, value, icon, segment, external, created_at, updated_at)
+      VALUES #{community_urls.join(",")}
+      RETURNING sidebar_urls.id
+    SQL
+
+    sidebar_section_links =
+      result.map.with_index do |url, index|
+        "(-1, #{url.id}, 'SidebarUrl', #{community_section.id}, #{index},  now(), now())"
+      end
+
+    DB.query <<~SQL
+      INSERT INTO sidebar_section_links(user_id, linkable_id, linkable_type, sidebar_section_id, position, created_at, updated_at)
+      VALUES #{sidebar_section_links.join(",")}
+    SQL
+  end
```

Comment:
I feel like this method should live in the `SidebarSection` model or some sort of service class instead of bloating up the controller.

Reply:
Also we probably want all the queries here to happen in the same transaction.